tests/headers/unsorted-items.h

```diff
@@ -0,0 +1,15 @@
+int foo();
+typedef int number;
+int bar(number x);
+struct Point
+{
+    number x;
+    number y;
```

Comment:
Is this test testing much? It seems it's not using the new feature, so maybe remove it?

Reply:
One could argue that this is testing that not using `--sort-semantically` doesn't change the order. But in a sense this is tested with the rest of the test suite already as we didn't modify any expected `rs` file.

Reply:
@emilio should we remove it then?

Reply:
Either way is fine, sorry for the lag.